### PR TITLE
Echoes database videos

### PR DESCRIPTION
**Pull request by MasierAzulite**
142 video links for Echoes tricks.

### CODE REVIEW

File: commit message/commit message.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Database video additions
+
+Added: 142 video links to the database for Echoes
```
Comment (dyceron) [inline commit message/commit message.txt:3]: I'm not sure where this file comes from, but it's not something we accept. I assume it's from your git program or whatever. Please remove it.

**Review by dyceron — DISMISSED** [review]
Don't forget to add a changelog entry for this.

File: commit message/commit message.txt (hunk 1 above)
Comment (MasierAzulite) [reply commit message/commit message.txt]: Accidentally added a file twice I think. It's gone now.

File: randovania/games/prime2/json_data/Agon Wastes.txt [hunk 2]
```diff
@@ -326,6 +328,9 @@ Extra - in_dark_aether: False
 > Door to Mine Shaft; Heals? False
   * Layers: default
   * Seeker Launcher Blast Shield (destructible with Screw Attack; Advanced) to Mine Shaft/Door to Mining Station B
+  * https://www.youtube.com/watch?v=-LZluWe8QvM&list=PLevfta9o_QKWuQNXAnmBXtzW4nmU5VLyN&index=46
```
[the remark this answers is not in the document]
Comment (toasterparty) [reply randovania/games/prime2/json_data/Agon Wastes.txt:331]: Honestly I think the exporter will strip the playlist reference here. Would have to double check. If so, then it's fine as is.

**Review by henriquegemignani — APPROVED** [review]

**Pull request merged**